disnake/ui/modal.py

```diff
@@ -47,38 +54,37 @@ class Modal:
 
     Parameters
     ----------
-    title: str
+    title: :class:`str`
         The title of the modal.
-    custom_id: str
+    custom_id: :class:`str`
         The custom ID of the modal.
-    components: List[:class:`~.ui.InputText`]
-        The components to display in the modal. Maximum of 5.
+    components: |components_type|
```

Comment:
Kind of related to my other message but **in my opinion**, it's better to just send components objects, because doing
```py
class Modal(ui.Modal):

     def __init__(self):
     
        components = [ui.ActionRow(
                         ui.InputText(...)
                     )]
        # etc
```
it's not that user friendly, and since you can only have 5 action rows, it wouldn't make sense, imo.

Reply:
You misunderstood the annotation

Reply:
See https://github.com/Victorsitou/disnake/pull/3#discussion_r791967301